pkg/operator/featureupgradablecontroller/feature_upgradeable_controller.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	featureGatesAllowingUpgrade = sets.NewString("")
+	featureGatesAllowingUpgrade = sets.NewString("", string(configv1.LatencySensitive))
```

Comment:
Can we change it to `featureGatesAllowingUpgrade = sets.NewString(string(configv1.LatencySensitive))`

Reply:
No, that actually fails one of the tests (I tried). The empty string seems to represent a situation with no feature gates.

Reply:
I had such feelings, thanks for the clarification.